[FR] Update `Medals/Unlock_requirements/Dedication`, `Medals/Unlock_requirements`, `Medals/Legacy_Beatmap_Packs`, `Medals` (#6410)
ppy/osu-wiki: wiki/Medals/fr.md

# Médailles

Les **médailles**, précédemment connues sous le nom de *réalisations*, sont des badges sur le profil d'un utilisateur qui indiquent certains objectifs qui ont été atteints par l'utilisateur. Actuellement, il y a **247** médailles à débloquer dans huit catégories différentes au total : [Beatmap Packs](#beatmap-packs), [Skill](#skill), [Dedication](#dedication), [Mod Introduction](#mod-introduction), [Hush-Hush](#hush-hush), [Beatmap Spotlights](#beatmap-spotlights), [Seasonal Spotlights](#seasonal-spotlights), et [Beatmap Challenge Packs](#beatmap-challenge-packs).

## Beatmap Packs

Les médailles **Beatmap Packs** sont des médailles attribuées après avoir terminé certains [Theme Beatmap Packs](https://osu.ppy.sh/beatmaps/packs?type=theme) ou [Artist/Album Beatmap Packs](https://osu.ppy.sh/beatmaps/packs?type=artist). Ces médailles peuvent être obtenues en terminant chaque beatmap au moins une fois dans n'importe quelle difficulté du beatmap pack correspondant.

> Tous les beatmaps doivent être réalisés dans le mode de jeu [osu!](/wiki/Game_mode/osu!) pour obtenir le badge, sauf indication contraire. L'utilisation de [mods de réduction de la difficulté](/wiki/Game_modifier#difficulté-réduite) ou de [mods d'augmentation de la difficulté](/wiki/Game_modifier#difficulté-augmenté) est autorisée.

Reviewer: ```suggestion
Toutes les beatmaps doivent être jouées dans le mode de jeu [osu!](/wiki/Game_mode/osu!) pour obtenir le badge, sauf indication contraire. L'utilisation de [mods de réduction de la difficulté](/wiki/Game_modifier#difficulté-réduite) ou de [mods d'augmentation de la difficulté](/wiki/Game_modifier#difficulté-augmenté) est autorisée.
```

> Toutes les beatmaps doivent être jouées dans le mode de jeu [osu!](/wiki/Game_mode/osu!) pour obtenir le badge, sauf indication contraire. L'utilisation de [mods de réduction de la difficulté](/wiki/Game_modifier#difficulté-réduite) ou de [mods d'augmentation de la difficulté](/wiki/Game_modifier#difficulté-augmenté) est autorisée.

Reviewer: ```suggestion
Toutes les beatmaps doivent être jouées dans le mode de jeu [osu!](/wiki/Game_mode/osu!) pour obtenir le badge, sauf indication contraire. L'utilisation de [mods réduisant la difficulté](/wiki/Game_modifier#difficulté-réduite) ou de [mods augmentant la difficulté](/wiki/Game_modifier#difficulté-augmentée) est autorisée.
```

> Il convient toutefois de noter que le contenu de tous les Packs Video Game Packs, Anime Packs, Internet! Packs et Rhythm Game Packs a été considérablement modifié depuis [leur première introduction en 2008](https://osu.ppy.sh/community/forums/topics/1853). Pour la liste de ces beatmaps packs dans leur état d'origine, voir [Anciens beatmaps packs](/wiki/Medals/Legacy_Beatmap_Packs).

Reviewer: ```suggestion
Il convient toutefois de noter que le contenu de tous les packs Video Game Packs, Anime Packs, Internet! Packs et Rhythm Game Packs a été considérablement modifié depuis [leur première introduction en 2008](https://osu.ppy.sh/community/forums/topics/1853). Pour la liste de ces beatmaps packs dans leur état d'origine, voir [Anciens beatmaps packs](/wiki/Medals/Legacy_Beatmap_Packs).
```

## Skill

> Les médailles **Skill** sont des médailles attribuées à la suite de divers accomplissements basés sur les compétences, que ce soit en atteignant certains jalons de [combo](/wiki/Beatmapping/Combo), en atteignant certains rangs dans le [classement global des points de performance](https://osu.ppy.sh/rankings/osu/performance), ou en réalisant un clear ou un [full combo](/wiki/Full_combo) sur n'importe quel beatmap [qualifiée](/wiki/Beatmap/Category#qualifiée), [classée](/wiki/Beatmap/Category#classée), ou [approuvée](/wiki/Beatmap/Category#approuvée) dans certaines fourchettes de [star rating](/wiki/Beatmapping/Star_rating).

Reviewer: ```suggestion
Les médailles **Skill** sont des médailles attribuées à la suite de divers accomplissements basés sur les compétences, que ce soit en atteignant certains seuils de [combo](/wiki/Beatmapping/Combo), en atteignant certains rangs dans le [classement global des points de performance](https://osu.ppy.sh/rankings/osu/performance), ou en réalisant un clear ou un [full combo](/wiki/Full_combo) sur n'importe quel beatmap [qualifiée](/wiki/Beatmap/Category#qualifiée), [classée](/wiki/Beatmap/Category#classée), ou [approuvée](/wiki/Beatmap/Category#approuvée) dans certaines fourchettes de [star rating](/wiki/Beatmapping/Star_rating).
```

> Les médailles **Skill** sont des médailles attribuées à la suite de divers accomplissements basés sur les compétences, que ce soit en atteignant certains seuils de [combo](/wiki/Beatmapping/Combo), en atteignant certains rangs dans le [classement global des points de performance](https://osu.ppy.sh/rankings/osu/performance), ou en réalisant un clear ou un [full combo](/wiki/Full_combo) sur n'importe quel beatmap [qualifiée](/wiki/Beatmap/Category#qualifiée), [classée](/wiki/Beatmap/Category#classée), ou [approuvée](/wiki/Beatmap/Category#approuvée) dans certaines fourchettes de [star rating](/wiki/Beatmapping/Star_rating).

Reviewer: ```suggestion
Les médailles **Skill** sont des médailles attribuées à la suite de divers accomplissements basés sur les compétences, que ce soit en atteignant certains seuils de [combo](/wiki/Beatmapping/Combo), en atteignant certains rangs dans le [classement global des points de performance](https://osu.ppy.sh/rankings/osu/performance), ou en réalisant un clear ou un [full combo](/wiki/Full_combo) sur n'importe quelle beatmap [qualifiée](/wiki/Beatmap/Category#qualifiée), [classée](/wiki/Beatmap/Category#classée), ou [approuvée](/wiki/Beatmap/Category#approuvée) dans certaines fourchettes de [star rating](/wiki/Beatmapping/Star_rating).
```

Toutes les médailles Mod Introduction peuvent être débloquées dans n'importe quel [mode de jeu](/wiki/Game_mode), sauf indication contraire.

## Hush-Hush

Les médailles **Hush-Hush** sont des médailles "secrètes" décernées pour des exploits très spécifiques, qui ne sont pas explicitement annoncés au joueur (d'où leur nom). Ces exploits peuvent aller de la réalisation d'une beatmap dans des circonstances particulières à l'obtention d'un score avec une combinaison de mods inhabituelle. Malgré leur nature secrète, il existe des indices pour aider tous les chasseurs de médailles curieux dans leurs recherches :

- Les noms des médailles elles-mêmes,
- Les brèves descriptions des médailles, et
- Les *textes flair* des médailles (s'il y en a), consultables en survolant les cartes correspondantes.

Comme les médailles Hush-Hush sont censées être des réalisations secrètes, les conditions de déverrouillage ne sont pas directement spécifiées ; cependant, les joueurs sont [invités à travailler ensemble pour trouver comment obtenir toutes les médailles Hush-Hush](https://osu.ppy.sh/home/news/2016-08-17-new-hush-hush-medals).

## Beatmap Spotlights

> Les médailles **Beatmap Spotlights** sont des médailles attribuées après avoir terminé certains beatmap pack publiés dans le cadre des [Monthly Beatmap Spotlights](https://osu.ppy.sh/home/news/2017-03-18-introducing-to-you-spotlights). Pour débloquer les médailles de cette catégorie, un joueur doit jouer toutes les maps *de l'un des quatre packs de mode de jeu* qui sont en compétition dans le classement Beatmap Spotlights correspondant. L'utilisation de [mods de réduction de la difficulté](/wiki/Game_modifier#difficulté-réduite) ou de [mods d'augmentation de la difficulté](/wiki/Game_modifier#difficulté-augmenté) est autorisée.

Reviewer: ```suggestion
Les médailles **Beatmap Spotlights** sont des médailles attribuées après avoir terminé certains beatmap packs publiés dans le cadre des [Monthly Beatmap Spotlights](https://osu.ppy.sh/home/news/2017-03-18-introducing-to-you-spotlights). Pour débloquer les médailles de cette catégorie, un joueur doit jouer toutes les maps *de l'un des quatre packs de mode de jeu* qui sont en compétition dans le classement Beatmap Spotlights correspondant. L'utilisation de [mods réduisant la difficulté](/wiki/Game_modifier#difficulté-réduite) ou de [mods augmentant la difficulté](/wiki/Game_modifier#difficulté-augmentée) est autorisée.
```

Bien que les Monthly Beatmap Spotlights aient été supprimés depuis avril 2018 au profit des [Seasonal Spotlights](https://osu.ppy.sh/home/news/2018-11-01-beatmap-spotlights-summer-2018), toutes les médailles qui s'y rapportent peuvent encore être obtenues à ce jour.

## Seasonal Spotlights

> Les médailles **Seasonal Spotlights** sont des médailles attribuées après avoir terminé certains beatmap pack publiés dans le cadre de la campagne [Seasonal Spotlights](https://osu.ppy.sh/home/news/2018-11-01-beatmap-spotlights-summer-2018). Pour débloquer les médailles de cette catégorie, un joueur doit jouer toutes les maps *de l'un des quatre packs de mode de jeu* qui sont en compétition dans le tableau de classement des Seasonal Spotlights correspondants. L'utilisation de [mods de réduction de la difficulté](/wiki/Game_modifier#difficulté-réduite) ou de [mods d'augmentation de la difficulté](/wiki/Game_modifier#difficulté-augmenté) est autorisée.

Reviewer: ```suggestion
Les médailles **Seasonal Spotlights** sont des médailles attribuées après avoir terminé certains beatmap packs publiés dans le cadre de la campagne [Seasonal Spotlights](https://osu.ppy.sh/home/news/2018-11-01-beatmap-spotlights-summer-2018). Pour débloquer les médailles de cette catégorie, un joueur doit jouer toutes les maps *de l'un des quatre packs de mode de jeu* qui sont en compétition dans le tableau de classement des Seasonal Spotlights correspondants. L'utilisation de [mods réduisant la difficulté](/wiki/Game_modifier#difficulté-réduite) ou de [mods augmentant la difficulté](/wiki/Game_modifier#difficulté-augmentée) est autorisée.
```

Bien que les Seasonal Spotlights aient été supprimés au début de l'année 2020 en faveur du [système actuel des Beatmap Spotlights](/wiki/Beatmap_Spotlights), toutes les médailles qui s'y rapportent peuvent encore être obtenues à ce jour.

## Beatmap Challenge Packs

> Les médailles **Beatmap Challenge Packs**, ou encore appelées [*Skill Packs medals*](https://osu.ppy.sh/home/news/2020-11-20-featured-artist-beatmap-updates-from-the-mappers-guild#pack-medal-updates), sont des médailles attribuées après avoir franchi le niveau de difficulté le plus élevé de toutes les beatmaps contenus dans certains [Theme Beatmap Packs](https://osu.ppy.sh/beatmaps/packs?type=theme) ou [Artist/Album Beatmap Packs](https://osu.ppy.sh/beatmaps/packs?type=artist) *sans [mods de réduction de la difficulté](/wiki/Game_modifier#difficulté-réduite) activés*. Ces beatmap pack sont spécialement conçus pour tester la compréhension de chaque joueur de divers aspects du jeu en fonction d'un niveau de compétence spécifique, par exemple dans une plage de [star ranting](/wiki/Beatmapping/Star_rating) ou de difficulté maximale.

Reviewer: ```suggestion
Les médailles **Beatmap Challenge Packs**, ou encore appelées [*Skill Packs medals*](https://osu.ppy.sh/home/news/2020-11-20-featured-artist-beatmap-updates-from-the-mappers-guild#pack-medal-updates), sont des médailles attribuées après avoir franchi le niveau de difficulté le plus élevé de toutes les beatmaps contenues dans certains [Theme Beatmap Packs](https://osu.ppy.sh/beatmaps/packs?type=theme) ou [Artist/Album Beatmap Packs](https://osu.ppy.sh/beatmaps/packs?type=artist) *sans [mods de réduction de la difficulté](/wiki/Game_modifier#difficulté-réduite) activés*. Ces beatmap packs sont spécialement conçus pour tester la compréhension de chaque joueur de divers aspects du jeu en fonction d'un niveau de compétence spécifique, par exemple dans une fourchette de [star rating](/wiki/Beatmapping/Star_rating) ou de difficulté maximale.
```

Toutes les médailles Mod Introduction peuvent être débloquées dans n'importe quel [mode de jeu](/wiki/Game_mode), sauf indication contraire.

## Hush-Hush

Les médailles **Hush-Hush** sont des médailles "secrètes" décernées pour des exploits très spécifiques, qui ne sont pas explicitement annoncés au joueur (d'où leur nom). Ces exploits peuvent aller de la réalisation d'une beatmap dans des circonstances particulières à l'obtention d'un score avec une combinaison de mods inhabituelle. Malgré leur nature secrète, il existe des indices pour aider tous les chasseurs de médailles curieux dans leurs recherches :

> 

Author: ```suggestion
```

- Les noms des médailles elles-mêmes,
- Les brèves descriptions des médailles, et
- Les *textes flair* des médailles (s'il y en a), consultables en survolant les cartes correspondantes.

Comme les médailles Hush-Hush sont censées être des réalisations secrètes, les conditions de déverrouillage ne sont pas directement spécifiées ; cependant, les joueurs sont [invités à travailler ensemble pour trouver comment obtenir toutes les médailles Hush-Hush](https://osu.ppy.sh/home/news/2016-08-17-new-hush-hush-medals).

## Beatmap Spotlights

> Les médailles **Beatmap Spotlights** sont des médailles attribuées après avoir terminé certains beatmap pack publiés dans le cadre des [Monthly Beatmap Spotlights] (https://osu.ppy.sh/home/news/2017-03-18-introducing-to-you-spotlights). Pour débloquer les médailles de cette catégorie, un joueur doit jouer toutes les maps *de l'un des quatre packs de mode de jeu* qui sont en compétition dans le classement Beatmap Spotlights correspondant. L'utilisation de [mods de réduction de la difficulté](/wiki/Game_modifier#difficulté-réduite) ou de [mods d'augmentation de la difficulté](/wiki/Game_modifier#difficulté-augmenté) est autorisée.

Author: ```suggestion
Les médailles **Beatmap Spotlights** sont des médailles attribuées après avoir terminé certains beatmap pack publiés dans le cadre des [Monthly Beatmap Spotlights](https://osu.ppy.sh/home/news/2017-03-18-introducing-to-you-spotlights). Pour débloquer les médailles de cette catégorie, un joueur doit jouer toutes les maps *de l'un des quatre packs de mode de jeu* qui sont en compétition dans le classement Beatmap Spotlights correspondant. L'utilisation de [mods de réduction de la difficulté](/wiki/Game_modifier#difficulté-réduite) ou de [mods d'augmentation de la difficulté](/wiki/Game_modifier#difficulté-augmenté) est autorisée.
```

Bien que les Monthly Beatmap Spotlights aient été supprimés depuis avril 2018 au profit des [Seasonal Spotlights](https://osu.ppy.sh/home/news/2018-11-01-beatmap-spotlights-summer-2018), toutes les médailles qui s'y rapportent peuvent encore être obtenues à ce jour.

## Seasonal Spotlights

> Les médailles **Seasonal Spotlights** sont des médailles attribuées après avoir terminé certains beatmap pack publiés dans le cadre de la campagne [Seasonal Spotlights] (https://osu.ppy.sh/home/news/2018-11-01-beatmap-spotlights-summer-2018). Pour débloquer les médailles de cette catégorie, un joueur doit jouer toutes les maps *de l'un des quatre packs de mode de jeu* qui sont en compétition dans le tableau de classement des Seasonal Spotlights correspondants. L'utilisation de [mods de réduction de la difficulté](/wiki/Game_modifier#difficulté-réduite) ou de [mods d'augmentation de la difficulté](/wiki/Game_modifier#difficulté-augmenté) est autorisée.

Author: ```suggestion
Les médailles **Seasonal Spotlights** sont des médailles attribuées après avoir terminé certains beatmap pack publiés dans le cadre de la campagne [Seasonal Spotlights](https://osu.ppy.sh/home/news/2018-11-01-beatmap-spotlights-summer-2018). Pour débloquer les médailles de cette catégorie, un joueur doit jouer toutes les maps *de l'un des quatre packs de mode de jeu* qui sont en compétition dans le tableau de classement des Seasonal Spotlights correspondants. L'utilisation de [mods de réduction de la difficulté](/wiki/Game_modifier#difficulté-réduite) ou de [mods d'augmentation de la difficulté](/wiki/Game_modifier#difficulté-augmenté) est autorisée.
```

> Bien que les Seasonal Spotlights aient été supprimés au début de l'année 2020 en faveur du [système actuel des Beatmap Spotlights] (/wiki/Beatmap_Spotlights), toutes les médailles qui s'y rapportent peuvent encore être obtenues à ce jour.

Author: ```suggestion
Bien que les Seasonal Spotlights aient été supprimés au début de l'année 2020 en faveur du [système actuel des Beatmap Spotlights](/wiki/Beatmap_Spotlights), toutes les médailles qui s'y rapportent peuvent encore être obtenues à ce jour.
```

## Beatmap Challenge Packs

> Les médailles **Beatmap Challenge Packs**, ou encore appelées [*Skill Packs medals*](https://osu.ppy.sh/home/news/2020-11-20-featured-artist-beatmap-updates-from-the-mappers-guild#pack-medal-updates), sont des médailles attribuées après avoir franchi le niveau de difficulté le plus élevé de toutes les beatmaps contenus dans certains [Theme Beatmap Packs](https://osu.ppy.sh/beatmaps/packs?type=theme) ou [Artist/Album Beatmap Packs](https://osu.ppy.sh/beatmaps/packs?type=artist) *sans [mods de réduction de la difficulté] (/wiki/Game_modifier#difficulté-réduite) activés*. Ces beatmap pack sont spécialement conçus pour tester la compréhension de chaque joueur de divers aspects du jeu en fonction d'un niveau de compétence spécifique, par exemple dans une plage de [star ranting](/wiki/Beatmapping/Star_rating) ou de difficulté maximale.

Author: ```suggestion
Les médailles **Beatmap Challenge Packs**, ou encore appelées [*Skill Packs medals*](https://osu.ppy.sh/home/news/2020-11-20-featured-artist-beatmap-updates-from-the-mappers-guild#pack-medal-updates), sont des médailles attribuées après avoir franchi le niveau de difficulté le plus élevé de toutes les beatmaps contenus dans certains [Theme Beatmap Packs](https://osu.ppy.sh/beatmaps/packs?type=theme) ou [Artist/Album Beatmap Packs](https://osu.ppy.sh/beatmaps/packs?type=artist) *sans [mods de réduction de la difficulté](/wiki/Game_modifier#difficulté-réduite) activés*. Ces beatmap pack sont spécialement conçus pour tester la compréhension de chaque joueur de divers aspects du jeu en fonction d'un niveau de compétence spécifique, par exemple dans une plage de [star ranting](/wiki/Beatmapping/Star_rating) ou de difficulté maximale.
```
